src/app/components/addresses/AddressHeader/components/AddressLink.tsx

```diff
@@ -0,0 +1,25 @@
+import styled from "styled-components"
+import { Link, Typography } from "@amsterdam/asc-ui"
+import to from "app/routing/utils/to"
+
+type Props = {
+  bagId: string
+  headingSize?: "h1" | "h2" | "span"
+  isHeader?: boolean
```

Comment:
I think we can rename `headingSize` to `as`. And then remove `isHeader` altogether?